aiobotocore/endpoint.py

```diff
@@ -49,8 +61,11 @@ async def convert_to_response_dict(http_response, operation_model):
     elif operation_model.has_event_stream_output:
         response_dict['body'] = http_response.raw
     elif operation_model.has_streaming_output:
-        length = response_dict['headers'].get('content-length')
-        response_dict['body'] = StreamingBody(http_response.raw, length)
+        if httpx and isinstance(http_response.raw, httpx.Response):
+            response_dict['body'] = http_response.raw
```

Comment:
should keep the `StreamingBody` as this is our own version so we can adapt as necessary

Reply:
I went into detail on this in https://github.com/aio-libs/aiobotocore/pull/1085#issuecomment-1929749830

But if we want to get rid of e.g. `aread` vs `read` differences maybe it could make sense to create a `HttpxStreamingBody` that tries to strike a middle ground